glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
On Sat, Aug 29, 2015 at 01:40:07AM -0700, Lai Jiangshan wrote:
> When the namespaces have been created, it implies that at least one
> process has been started. (it is the `runc init` process for the
> runc implementation for example)

RunC currently conflates the container creation and application
execution, but that's not required.  For example, bocker:

1. creates a network namespace, cgroups, and networking here [1],
2. enters that container and executes the application [2], and
3. deletes the container [3,4]

It doesn't split those into separate commands (since it's based on the
Docker UI), but you *could* certainly split them into separate
commands, and that seems like the most sane approach to me.

[1]: https://github.com/p8952/bocker/blob/000633061c6cfcb99c8d9eef5aa483a44318f3e6/bocker#L62-L81
[2]: https://github.com/p8952/bocker/blob/000633061c6cfcb99c8d9eef5aa483a44318f3e6/bocker#L82-L87
[3]: https://github.com/p8952/bocker/blob/000633061c6cfcb99c8d9eef5aa483a44318f3e6/bocker#L88-L89
[4]: https://github.com/p8952/bocker/blob/000633061c6cfcb99c8d9eef5aa483a44318f3e6/bocker#L41